hadoop-ozone/dev-support/checks/unit.sh

```diff
@@ -16,4 +16,4 @@
 
 DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" >/dev/null 2>&1 && pwd )"
 CHECK=unit
-source "${DIR}/junit.sh" -pl \!:ozone-integration-test,\!:mini-chaos-tests -DexcludedGroups=native "$@"
+source "${DIR}/junit.sh" -pl \!:ozone-integration-test,\!:mini-chaos-tests -DexcludedGroups=native | unhealthy "$@"
```

Comment:
Need to quote the parameter value, as `|` is a special symbol for the shell, too.  And the category is needed for JUnit4.

```suggestion
source "${DIR}/junit.sh" -pl \!:ozone-integration-test,\!:mini-chaos-tests -DexcludedGroups="native | unhealthy | org.apache.ozone.test.UnhealthyTest" "$@"
```

Reply:
Thanks @adoroszlai for proactively reviewing it. Handled the above issue.